tests/ccm.tests.ps1

```diff
@@ -0,0 +1,75 @@
+[CmdletBinding()]
+Param(
+    [Parameter(Mandatory)]
+    [String]
+    $Fqdn
+)
+
+Describe "Chocolatey Central Management Configuration" {
+    Context "Services" {
+        BeforeAll {
+            function Get-RemoteCertificate {
+                param(
+                    [Alias('CN')]
+                    [Parameter(Mandatory = $true, Position = 0)]
+                    [string]$ComputerName,
+            
+                    [Parameter(Position = 1)]
+                    [UInt16]$Port = 8443
+                )
+            
+                $tcpClient = New-Object System.Net.Sockets.TcpClient($ComputerName, $Port)
+                $sslProtocolType = [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.SecurityProtocolType]::Tls12
+                try {
+                    $tlsClient = New-Object System.Net.Security.SslStream($tcpClient.GetStream(), 'false', $callback)
+                    $tlsClient.AuthenticateAsClient($ComputerName, $null, $sslProtocolType, $false)
+            
+                    return $tlsClient.RemoteCertificate -as [System.Security.Cryptography.X509Certificates.X509Certificate2]
+                }
+                finally {
+                    if ($tlsClient -is [IDisposable]) {
+                        $tlsClient.Dispose()
+                    }
+            
+                    $tcpClient.Dispose()
```

Comment:
Is there a reason we're checking for `$tlsClient` to be disposable, but not `$tcpClient`? On that note, should the try be a few lines higher to capture the `$tcpClient` declaration?